components/filters/RadiusRangeFilter.vue

```diff
@@ -76,6 +43,7 @@ watch([() => props.filter.searchRadius, () => props.filter.zipcode], ([newRadius
   validateRadiusAndZipcode(newRadius, newZipcode)
 }, { immediate: true })
 
+// Validate if the user has inputed a zip code along with the radius as the user location is required
```

Comment:
The comment tells me that this function validates the input and checks if there is a ziip code ALONG with the radius => the real check tests if there is a radius and no zip code